modules/browser_object_menu_bar.py

```diff
@@ -0,0 +1,24 @@
+from selenium.webdriver import ActionChains
+
+from modules.page_base import BasePage
+
+
+class MenuBar(BasePage):
+    """Page Object Model for Menu Bar navigation"""
+
+    def activate_menu_bar(self) -> BasePage:
+        """Enables the Menu Bar at the top of the window (if needed)"""
+        if self.sys_platform() != "Darwin":
+            with self.driver.context(self.driver.CONTEXT_CHROME):
+                element = self.get_element("toolbar-blank-space")
```

Comment:
`BasePage` already has a function for this, you can replace lines 13-15 with `self.context_click("toolbar-blank-space")`